Added example with column defined as a string (#1139)
yiisoft/db: docs/guide/en/command/ddl.md

)->execute();
```

> You can pass a description of the column structure as a string

Reviewer: ```suggestion
You can pass a column definition as a string:
```
